packages/ui/src/theme/css/component/_fieldControl.scss

```diff
@@ -71,16 +71,6 @@
     }
   }
 
-  // for disabled styles
```

Comment:
These disabled styles are moved further down the file to fix https://github.com/aws-amplify/amplify-ui/issues/4409